src/Sarif.Multitool.FunctionalTests/SarifMultitoolTestBase.cs

```diff
@@ -57,13 +57,13 @@ private static void SelectiveCompare(SarifLog actualLog, SarifLog expectedLog)
 
             SelectiveCompare(actualResults, expectedResults);
 
-            Notification[] actualConfigurationNotifications = SafeListToArray(actualLog.Runs[0].ConfigurationNotifications);
-            Notification[] expectedConfigurationNotifications = SafeListToArray(expectedLog.Runs[0].ConfigurationNotifications);
+            Notification[] actualConfigurationNotifications = SafeListToArray(actualLog.Runs[0].Invocation.ConfigurationNotifications);
```

Comment:
Again make sure `Invocation` exists. (It's test code, so maybe you actually do know that for sure.)